cmd/openshift-install/version.go

```diff
@@ -28,5 +30,8 @@ func runVersionCmd(cmd *cobra.Command, args []string) error {
 	if image, err := bootstrap.DefaultReleaseImage(); err == nil {
 		fmt.Printf("release image %s\n", image)
 	}
+	if rhcosmeta, err := rhcos.FetchRHCOSBuild(context.TODO()); err == nil {
+		fmt.Printf("RHEL CoreOS bootimage %s\n", rhcosmeta.OSTreeVersion)
```

Comment:
It seems like we are required to use the full name, `Red Hat Enterprise Linux CoreOS`.
cc @kalexand-rh

Reply:
I agree. I fixed the CloudFormation templates on https://github.com/openshift/installer/pull/1794. Thank you for changing the other instances.